guides/source/form_helpers.md

```diff
@@ -17,9 +17,9 @@ After reading this guide, you will know:
 
 --------------------------------------------------------------------------------
 
-NOTE: This guide is not intended to be a complete documentation of available form helpers and their arguments. Please visit [the Rails API documentation](https://api.rubyonrails.org/classes/ActionView/Helpers.html) for a complete reference of all available helpers.
+NOTE: This guide is not intended to be a complete list of all available form helpers. Please reference [the Rails API documentation](https://api.rubyonrails.org/classes/ActionView/Helpers.html) for an exhaustive list of form helpers and their arguments.
```

Comment:
Can we use "Please see"or "Please refer to" rather than reference?

Reply:
Also maybe replace `exhaustive` with the simpler `complete`?
```suggestion
NOTE: This guide is not intended to be a complete list of all available form helpers. Please reference [the Rails API documentation](https://api.rubyonrails.org/classes/ActionView/Helpers.html) for a complete list of form helpers and their arguments.
```